src/tools/utils/index.tsx

```diff
@@ -59,39 +59,39 @@ export const getRemoteIDFromResponse = (
 
 const idMappingStatusPattern = /\/idmapping(\/(?<idMappingTarget>\w+))?\/results\/[a-f\d]/;
 const statuses = Object.values(Status);
-const peptideSearchStatusPattern = new RegExp(
-  `Job status: (?<status>${statuses.join('|')})`
-);
 
-export const getStatusFromResponse = (
+export const getStatusFromResponse = async (
   jobType: JobTypes,
-  response: AxiosResponse<Status | { jobStatus: Status } | string>
-): [status: Status, idMappingTarget?: IDMappingNamespace] => {
+  response: Response
+  // response: Status | { jobStatus: Status } | string
```

Comment:
```suggestion
```

Reply:
Indeed this can be removed, sorry that this slipt through